app/helpers/course/controller_helper.rb

```diff
@@ -30,6 +30,16 @@ def display_user(user)
     end
   end
 
+  # URL to the profile page of the given +CourseUser+ in the current course
+  #
+  # @param [CourseUser] course_user The +CourseUser+ to link to
+  # @return [String] A URL that points to the +CourseUser+ profile page
+  def url_to_course_user(course_user)
```

Comment:
I dont think you need to have this helper at all since we know that `course_user` is `CourseUser`, and the link can just be generated by calling `course_user_path` directly

Reply:
Perhaps, you can also pass `user` to this function. Check if course_user if nil. If it is nil, then return user_path instead. Call this something like `url_to_course_user_or_user`